samples/ipc/ipc_service/Kconfig.sysbuild

```diff
@@ -8,6 +8,3 @@ source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
 
 config REMOTE_BOARD
 	string "The board used for remote target"
-	default "nrf5340dk_nrf5340_cpunet" if $(BOARD) = "nrf5340dk_nrf5340_cpuapp"
-	default "nrf5340dk_nrf5340_cpunet" if $(BOARD) = "nrf5340dk_nrf5340_cpuapp_ns"
-	default "UNKNOWN"
```

Comment:
Maybe keep `default "UNKNOWN"` to get a less misleading error when this is not set.
Alternatively, update `sysbuild.cmake` to show a helpful error `if("${SB_CONFIG_REMOTE_BOARD}" STREQUAL "")`.